project/Dependencies.scala

```diff
@@ -16,37 +16,38 @@ object Dependencies {
 
   object V {
     // Scala (Loader)
-    val decline          = "2.1.0"
-    val igluClient       = "1.3.0"
-    val igluCore         = "1.0.0"
-    val badrows          = "2.1.0"
+    val decline          = "2.4.1"
+    val igluClient       = "3.0.0-M1"
```

Comment:
I'm aware this is unusual to depend on a `-M1`  version.  But we're still finalising what the 3.0.0 public interface will look like for iglu-scala-client.  I don't want to delay the rdb-loader release while we still figure it out.

I'm comfortable with releasing this against 3.0.0-M1 for now, and bumping it to 3.0.0 in the next rdb-loader release.

Reply:
I'm fine with being pragmatic